instrumentation/opentelemetry-instrumentation-sklearn/pyproject.toml

```diff
@@ -19,9 +19,6 @@ classifiers = [
   "Programming Language :: Python",
   "Programming Language :: Python :: 3",
   "Programming Language :: Python :: 3.8",
-  "Programming Language :: Python :: 3.9",
-  "Programming Language :: Python :: 3.10",
-  "Programming Language :: Python :: 3.11",
```

Comment:
Why remove these?

Reply:
Because the sklearn package it traces doesn't work with that python versions

Reply:
I see, because we only support 0.X: https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation/opentelemetry-instrumentation-sklearn/pyproject.toml#L33